### ISSUE TITLE
求环境

### ISSUE THREAD

**Issue by Eiven575** [weizhepei/HBT#80]
可以提供一份能够复现论文中精度的requirement.txt吗？

**Comment by xiuzhilalala** [comment]
absl-py==1.0.0
astor==0.8.1
cached-property==1.5.2
colorama==0.4.4
gast==0.5.3
grpcio==1.44.0
h5py==3.6.0
importlib-metadata==4.11.3
Keras==2.2.4
Keras-Applications==1.0.8
keras-bert==0.80.0
keras-embed-sim==0.10.0
keras-layer-normalization==0.15.0
keras-multi-head==0.28.0
keras-pos-embd==0.12.0
keras-position-wise-feed-forward==0.7.0
Keras-Preprocessing==1.1.2
keras-self-attention==0.51.0
keras-transformer==0.33.0
Markdown==3.3.6
mock==4.0.3
numpy==1.21.5
protobuf==3.19.4
PyYAML==6.0
scipy==1.7.3
six==1.16.0
tensorboard==1.13.1
tensorflow==1.13.1
tensorflow-estimator==1.13.0
tensorflow-gpu==1.13.1
termcolor==1.1.0
tqdm==4.63.0
typing-extensions==4.1.1
Werkzeug==2.0.3
zipp==3.7.0

**Comment by nlper01** [comment]
> absl-py==1.0.0 astor==0.8.1 cached-property==1.5.2 colorama==0.4.4 gast==0.5.3 grpcio==1.44.0 h5py==3.6.0 importlib-metadata==4.11.3 Keras==2.2.4 Keras-Applications==1.0.8 keras-bert==0.80.0 keras-embed-sim==0.10.0 keras-layer-normalization==0.15.0 keras-multi-head==0.28.0 keras-pos-embd==0.12.0 keras-position-wise-feed-forward==0.7.0 Keras-Preprocessing==1.1.2 keras-self-attention==0.51.0 keras-transformer==0.33.0 Markdown==3.3.6 mock==4.0.3 numpy==1.21.5 protobuf==3.19.4 PyYAML==6.0 scipy==1.7.3 six==1.16.0 tensorboard==1.13.1 tensorflow==1.13.1 tensorflow-estimator==1.13.0 tensorflow-gpu==1.13.1 termcolor==1.1.0 tqdm==4.63.0 typing-extensions==4.1.1 Werkzeug==2.0.3 zipp==3.7.0

你好，请问对应的Python版本是多少

**Comment by YakiChen** [comment]
> ABSL-PY==1.0.0 ASTOR==0.8.1 缓存属性==1.5.2 COLORAMA==0.4.4 GAST==0.5.3 GRPCIO==1.44.0 H5PY==3.6.0 importlib-metadata==4.11.3 Keras==2.2.4 Keras-Applications==1.0.8 Keras-BERT==0.80.0 Keras-embed-sim==0.10.0 Keras-Layer-Normalization==0.15.0 Keras-Multi-Head==0.28.0 Keras-POS-Embed==0.12.0keras-position-wise-feed-forward==0.7.0 Keras-预处理==1.1.2 keras-self-attention==0.51.0 keras-transformer==0.33.0 Markdown==3.3.6 mock==4.0.3 numpy==1.21.5 protobuf==3.19.4 PyYAML==6.0 scipy==1.7.3 six==1.16.0 tensorboard==1.13.1 tensorflow==1.13.1 tensorflow-estimator==1.13.0 tensorflow-gpu==1.13.1 termcolor==1.1.0 tqdm==4.63.0 typing-extensions==4.1.1 Werkzeug==2.0.3 zipp==3.7.0

你好，请问你跑起来了嘛？看着你这没有这个codecs，是不需要也行嘛？
![image](https://github.com/user-attachments/assets/16ac95d7-c6dd-4b24-8f34-98611d55a7d0)
